files/build_templates/docker_image_ctl.j2

```diff
@@ -74,11 +74,7 @@ function preStartAction()
     if [ "$DATABASE_TYPE" != "chassisdb" ]; then
         if [[ ("$BOOT_TYPE" == "warm" || "$BOOT_TYPE" == "fastfast")  && -f $WARM_DIR/dump.rdb ]]; then
             # Load redis content from /host/warmboot/dump.rdb
-            docker cp $WARM_DIR/dump.rdb database$DEV:/var/lib/redis/dump.rdb
-        else
-            # Create an emtpy file and overwrite any RDB if already there
-            echo -n > /tmp/dump.rdb
-            docker cp /tmp/dump.rdb database$DEV:/var/lib/redis/
+            docker cp $WARM_DIR/dump.rdb database$DEV:/var/lib/dump.rdb
```

Comment:
>/var/lib/ [](start = 54, length = 9)

This directory seems strange.

Reply:
redis instance rdb file is at /var/lib/instance_name/, so I thought /var/lib is OK for temporary global location to hold rdb file. I will be deleted later when flushing db. 
BTW, what directory you think is good, /etc/sonic/ or  some directory originally exising in database docker?

Reply:
Better to keep old code, ie, still using `/var/lib/redis/dump.rdb`. Inside docker, you can copy it on-demand. If there is only 1 instance, no need to copy at all.

---
In reply to: [520934863](https://github.com/Azure/sonic-buildimage/pull/5773#discussion_r520934863) [](ancestors = 520934863)

Reply:
kept the old directory as dicussed